content-resources/src/main/java/flyway/sample/V1_0_12__add_download_basket_to_views.java

```diff
@@ -0,0 +1,49 @@
+package flyway.downloadbasket;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+
+import fi.nls.oskari.map.view.ViewService;
+import fi.nls.oskari.map.view.ViewServiceIbatisImpl;
+
+public class V1_0_12__add_download_basket_to_views implements JdbcMigration {
+	
+	private static final ViewService VIEW_SERVICE = new ViewServiceIbatisImpl();
+	private static final  String DOWNLOAD_BASKET = "download-basket";
+	
+	public void migrate(Connection connection) throws Exception {
+		long viewId = VIEW_SERVICE.getDefaultViewId();
```

Comment:
Note that this will only add the bundle to the system default view. Not to any role-specific views or views that users have saved. Are there any layers in the sample app that can be used with this bundle? Here's another example for doing the same thing, but with less code: https://github.com/oskariorg/oskari-server/blob/master/content-resources/src/main/java/flyway/sample/V1_0_8__add_statsgrid_to_default_views.java